.ci/containers/downstream-builder/generate_downstream.sh

```diff
@@ -118,16 +118,6 @@ if [ "$REPO" == "terraform-google-conversion" ]; then
     popd
 
     bundle exec compiler.rb -a -e terraform -f validator -o $LOCAL_PATH/tfplan2cai -v $VERSION
-
-    pushd $LOCAL_PATH
-
-    if [ "$COMMAND" == "downstream" ]; then
-      go get -d github.com/hashicorp/terraform-provider-google-beta@$BASE_BRANCH
```

Comment:
This line will need to stay, otherwise the downstream branch won't update terraform-provider-google-beta to latest in the `go.sum`